WalletWasabi.Fluent/ViewModels/Wallets/Send/TransactionPreviewViewModel.cs

```diff
@@ -314,6 +314,24 @@ private async Task<bool> InitialiseTransactionAsync()
 
 			return result ? await BuildTransactionAsync(reason) : null;
 		}
+		catch (ArgumentException ex)
+		{
+			Logger.LogInfo(ex.Message);
+
+			await ShowErrorAsync("Transaction Building", ex.ToUserFriendlyString(),
+				"Wasabi was unable to create your transaction.");
+
+			return null;
+		}
+		catch (NoAvailableCoinsException ex)
+		{
+			Logger.LogInfo(ex.Message);
+
+			await ShowErrorAsync("Transaction Building", ex.ToUserFriendlyString(),
+				"Wasabi was unable to create your transaction.");
+
+			return null;
+		}
```

Comment:
How did you get these exceptions?

Reply:
Well, they come from `TransactionFactory` 
https://github.com/zkSNACKs/WalletWasabi/blob/f963ea34e0b656e3e09e4da447b3e10746a3d5d2/WalletWasabi/Blockchain/Transactions/TransactionFactory.cs#L77

https://github.com/zkSNACKs/WalletWasabi/blob/f963ea34e0b656e3e09e4da447b3e10746a3d5d2/WalletWasabi/Blockchain/Transactions/TransactionFactory.cs#L106

They actually don't need their own catch block, I just didn't want them to use `LogError`.
Ofc, the simplest would be to let them run into the `catch (Exception ex)`.

Reply:
Why does `LogError` not suit here? For the sake of` info` instead of `error` it led to many code duplications. It doesn't seem good. 
At this point, if there is any error that stopped the transaction building (and is not handled) is a serious problem. Error is fine.